app/controllers/notes_controller.rb

```diff
@@ -28,7 +28,7 @@ def create
     if @note.save
       record_deed
       render json: {
-        html: render_to_string(partial: 'note.html', locals: { note: @note }, formats: [:html]),
+        html: render_to_string(partial: 'note', locals: { note: @note }, formats: [:html]),
```

Comment:
We pass in formats: [:html] instead of 'note.html' as that syntax is not supported anymore.